frontend/public/components/dashboard/dashboards-page/overview-dashboard/utilization-card.tsx

```diff
@@ -125,6 +143,9 @@ const UtilizationCard_: React.FC<DashboardItemProps & WithFlagsProps> = ({
           isLoading={!cpuUtilization}
           humanizeValue={humanizePercentage}
           query={queries[OverviewQuery.CPU_UTILIZATION]}
+          PopupComponent={ConsumerPopover}
+          humanizePopup={humanizeSeconds}
```

Comment:
you need to convert the value from prometheus to nanoseconds first. See these humanize functions https://github.com/openshift/console/blob/master/frontend/packages/console-shared/src/components/dashboard/top-consumers-card/ConsumersFilter.tsx#L17 for the given top consumer